### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by wandyezj**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,11 +45,10 @@ Also, Outlook add-ins do not support hosting Office.js offline due to a network
...
-
+> npm install @microsoft/office-js --save
 <br />
 
-Our policy requires that developers always reference the latest version of Office.js library. This is done to ensure that essential product updates are served quickly to the Office add-ins by always referencing the latest release of the library for a given version, such as Generally Available (GA) version. Given that the latest Office.js release is backward-compatible with prior releases, it is safe to update to the most recent release of the library when one is available. Hence, only the latest version of Office.js NPM package is made available for installation. 
+Our policy requires that developers always reference the latest version of Office.js library. This is done to ensure that essential product updates are served quickly to the Office add-ins by always referencing the latest release of the library for a given version, such as Generally Available (GA) version. Given that the latest Office.js release is backward-compatible with prior releases, it is safe to update to the most recent release of the library when one is available. Hence, only the latest version of Office.js NPM package is made available for installation.
```
Comment (lindalu-MSFT) [inline README.md]:
```suggestion
Our policy requires that developers always reference the latest version of Office.js library. This is to ensure that essential product updates are served quickly to the Office Add-ins by always referencing the latest release of the library for a given version, such as Generally Available (GA) version. Given that the latest Office.js release is backward-compatible with prior releases, it is safe to update to the most recent release of the library when one is available. Hence, only the latest version of Office.js NPM package is made available for installation.
```

File: README.md [hunk 2]
```diff
@@ -61,88 +60,84 @@ Installing the NPM package locally creates a set of static Office.js files in th
...
+  - DefinitelyTyped: <https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/office-js-preview/index.d.ts>
+  - @types: `@types/office-js-preview`.  (Acquire as `npm install @types/office-js-preview --save-dev`)
+- For **any** version:
+  - Inside of the NPM package, under `dist/office.d.ts`
+  - In this repo: [dist/office.d.ts](dist/office.d.ts)
 
 ### Using TypeScript definitions with the NPM package
 
-1. If you are using the Office.js NPM package for the [firewall scenario](#target-scenarios) and want a d.ts file that precisely matches the JS contents, use the d.ts file that is located within the `/dist/office.d.ts` folder of the NPM package. You can achieve this by using a [triple-slash reference](https://www.typescriptlang.org/docs/handbook/triple-slash-directives.html). 
+If you are using the Office.js NPM package for the [firewall scenario](#target-scenarios) and want a d.ts file that precisely matches the JS contents, use the d.ts file that is located within the `/dist/office.d.ts` folder of the NPM package. You can achieve this by using a [triple-slash reference](https://www.typescriptlang.org/docs/handbook/triple-slash-directives.html).
```
Comment (lindalu-MSFT) [inline README.md]:
```suggestion
If you're using the Office.js NPM package for the [firewall scenario](#target-scenarios) and want a d.ts file that precisely matches the JS contents, use the d.ts file that is located within the `/dist/office.d.ts` folder of the NPM package. Achieve this by using a [triple-slash reference](https://www.typescriptlang.org/docs/handbook/triple-slash-directives.html). 
```
This is per Microsoft Voice and Style guide:
1. Project friendliness
2. Revise weak wrting
https://styleguides.azurewebsites.net/Styleguide/Read?id=2696&topicid=29324
or https://docs.microsoft.com/en-us/style-guide/top-10-tips-style-voice

File: README.md [hunk 3]
```diff
@@ -61,88 +60,84 @@ Installing the NPM package locally creates a set of static Office.js files in th
...
+  - DefinitelyTyped: <https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/office-js/index.d.ts>
+  - @types: `@types/office-js`.  (Acquire as `npm install @types/office-js --save-dev`)
+- For latest **BETA** version of Office.js:
+  - DefinitelyTyped: <https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/office-js-preview/index.d.ts>
+  - @types: `@types/office-js-preview`.  (Acquire as `npm install @types/office-js-preview --save-dev`)
+- For **any** version:
+  - Inside of the NPM package, under `dist/office.d.ts`
+  - In this repo: [dist/office.d.ts](dist/office.d.ts)
 
 ### Using TypeScript definitions with the NPM package
```
Comment (lindalu-MSFT) [inline README.md]: We should probably fix the use of gerund verbs in headings at some point!

File: README.md [hunk 4]
```diff
@@ -61,88 +60,84 @@ Installing the NPM package locally creates a set of static Office.js files in th
...
 
-1. If you are using the Office.js NPM package for the [firewall scenario](#target-scenarios) and want a d.ts file that precisely matches the JS contents, use the d.ts file that is located within the `/dist/office.d.ts` folder of the NPM package. You can achieve this by using a [triple-slash reference](https://www.typescriptlang.org/docs/handbook/triple-slash-directives.html). 
+If you are using the Office.js NPM package for the [firewall scenario](#target-scenarios) and want a d.ts file that precisely matches the JS contents, use the d.ts file that is located within the `/dist/office.d.ts` folder of the NPM package. You can achieve this by using a [triple-slash reference](https://www.typescriptlang.org/docs/handbook/triple-slash-directives.html). 
 
-   - **Tip**: If you create a `references.ts` file at the root of the project, you can simply point the reference to `office.d.ts` there.
+- **Tip**: If you create a `references.ts` file at the root of the project, you can simply point the reference to `office.d.ts` there.
```
Comment (lindalu-MSFT) [inline README.md:86]:
```suggestion
- > [!TIP]
> If you create a `references.ts` file at the root of the project, simply point the reference to `office.d.ts` there.
```

File: README.md [hunk 5]
```diff
@@ -61,88 +60,84 @@ Installing the NPM package locally creates a set of static Office.js files in th
...
 
-2. If you are using the Office.js NPM package for **beta**, follow the guidance outlined in the preceding point (#1), but make sure to update often.
-
-If neither of these points applies to your scenario, you can just obtain the TypeScript definitions by using `@types/office-js` and reference the Office.js CDN at <https://appsforoffice.microsoft.com/lib/1/hosted/office.js> -- in which case, you don't need to use the Office.js NPM package.
+If you do not need the firewall scenario, obtain the TypeScript definitions by using `@types/office-js` and reference the Office.js CDN at <https://appsforoffice.microsoft.com/lib/1/hosted/office.js>. You don't need to use the Office.js NPM package.
```
Comment (lindalu-MSFT) [inline README.md]:
```suggestion
If you don't need the firewall scenario, obtain the TypeScript definitions by using `@types/office-js` and reference the Office.js CDN at <https://appsforoffice.microsoft.com/lib/1/hosted/office.js>. You don't need to use the Office.js NPM package.
```

File: README.md [hunk 6]
```diff
@@ -61,88 +60,84 @@ Installing the NPM package locally creates a set of static Office.js files in th
...
 
-<br />
-
 ## Accessing the NPM files via a CDN
 
-In addition to downloading the files locally, you can also use them via an external service like <https://unpkg.com>, which provides best-effort (no uptime guarantees) CDN hosting for npm packages.  This is especially useful for trying out alpha or beta builds.  To do so, simply change the script reference to:
-
-    <script src="https://unpkg.com/@microsoft/office-js/dist/office.js"></script>
+In addition to downloading the files locally, they may be used via an external service like <https://unpkg.com>, which provides best-effort (no uptime guarantees) CDN hosting for npm packages.  This is especially useful for trying out custom builds. To do so, simply change the script reference to:
```
Comment (lindalu-MSFT) [inline README.md]:
```suggestion
In addition to downloading the files locally, they may be used via an external service like <https://unpkg.com>, which provides best-effort (no uptime guarantees) CDN hosting for npm packages. This is especially useful for trying out custom builds. To do so, simply change the script reference to:
```

File: README.md [hunk 7]
```diff
@@ -61,88 +60,84 @@ Installing the NPM package locally creates a set of static Office.js files in th
...
 
-You can see the different versions of the NPM package listed in the dropdown on the top right at <https://unpkg.com/@microsoft/office-js/>.  This provides the alpha and beta versions as well.
+```html
+<script src="https://unpkg.com/@microsoft/office-js/dist/office.js"></script>
+```
 
-To view the latest version numbers for each of the tags, you can also run the following command on the command-line:
+The different versions of the NPM package are listed in the dropdown on the top right at <https://unpkg.com/@microsoft/office-js/>.
```
Comment (lindalu-MSFT) [inline README.md]:
```suggestion
The different versions of the NPM package are listed  on the top right in the dropdown list at <https://unpkg.com/@microsoft/office-js/>.
```

**Review by lindalu-MSFT — DISMISSED** [review]
Hi Joe, a few editorial comments! Pretty standard stuff, if you have any questions, LMK!

File: README.md (hunk 4 above)
Comment (wandyezj) [inline README.md]: What does > [!TIP] do?

note: this readme is used for the NPM website as well: https://www.npmjs.com/package/@microsoft/office-js

Comment (lindalu-MSFT) [reply README.md]: Never mind, that's for docs. It formats the markdown. It shouldn't be applied here, my fault.

File: README.md (hunk 3 above)
Comment (wandyezj) [reply README.md]: fixed

**Review by lindalu-MSFT — APPROVED** [review]

**Pull request merged**